app/lib/common/widget/CrossSigning.dart

```diff
@@ -215,30 +215,81 @@ class CrossSigning {
             ),
             child: Column(
               children: [
-                Text(
-                  _eventMap[txnId]!.verifyingThisDev
-                      ? AppLocalizations.of(context)!.verifyThisSession
-                      : AppLocalizations.of(context)!.verifySession,
-                  style: CrossSigningSheetTheme.primaryTextStyle,
+                Row(
+                  children: [
+                    Container(
+                      padding: const EdgeInsets.all(10.0),
+                      child: SvgPicture.asset(
+                        'assets/images/baseline-devices.svg',
+                      ),
+                    ),
+                    const SizedBox(width: 5),
+                    Text(
+                      AppLocalizations.of(context)!.verifyThisSession,
+                      style: CrossSigningSheetTheme.primaryTextStyle,
+                    ),
+                    const Spacer(),
+                    Padding(
+                      padding: const EdgeInsets.only(right: 10.0),
+                      child: IconButton(
+                        icon: const Icon(Icons.close),
+                        onPressed: () async {
+                          await event.cancelVerificationRequest();
+                          Get.back();
+                          _eventMap.remove(txnId);
+                        },
+                        color: Colors.white,
+                      ),
+                    ),
+                  ],
                 ),
-                Text(
-                  AppLocalizations.of(context)!.verificationScanSelfNotice,
-                  style: CrossSigningSheetTheme.secondaryTextStyle,
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(15.0, 8.0, 15, 8.0),
```

Comment:
You may change like this snippet.
`const EdgeInsets.symmetric(horizontal: 15.0, vertical: 8.0)`